src/content/en/fundamentals/engage-and-retain/push-notifications/index.markdown

```diff
@@ -1,24 +1,77 @@
 ---
 layout: shared/narrow-pages-list
-title: "Push Notifications"
-description: "You may already know how to implement push notifications. Are you using them well? Could you be using them better?"
-published_on: 2016-01-05
-updated_on: 2016-01-05
+title: "Push Notifications: Timely, Relevant, and Precise"
+description: "Push notifications are one of the most valuable capabilities of native apps, and this capability is now available on the web. To get the most out of them, notifications need to be timely, precise, and relevant."
+published_on: 2016-07-01
+updated_on: 2016-07-01
 order: 1
-translation_priority: 0
+translation_priority: 1
 authors:
   - josephmedley
 ---
 
-<p class="intro">
-  You may already know <a href="/web/fundamentals/primers/push-notifications">how to implement push notifications</a>. Are you using them well? Could you be using them better? 
+<p class="intro">Don't annoy your users, or you'll lose access to them
+	forever. Do we need to say anything more than that? We do because it's easier said than
+done.
+
```

Comment:
Because this is in a `<p>...</p>` tag, the line break is lost, probably a good idea to include `<br><br>` in here to split it up.